### DISCUSSION TITLE
[feature request] Continuous battle mode

### DISCUSSION THREAD

**Discussion by Kuroteba** [ikemen-engine/Ikemen-GO#922] [Feature Request]
I am now wishing that Swiss Army Knife's Continuous battle mode feature was in IKEMEN GO.
This will make it easier to adjust the strength of the characters when I want to hold a MUGEN tournament with other CPUs on IKEMEN GO.

Concrete ideas for additional functions

Continuous battle mode with high speed mode and frameskipping turned on, and output the results to a text file after all battles are over.

**Comment by Windblade-GR01** [comment]
There is already a feature like that called "randomtest" that outputs the result to a txt file.
You could also use launch parameters to set the engine to run as fast as it could.

**Comment by Kuroteba** [comment]
The "randomtest" does not allow me to select a combination of characters or colors to play with. Also, unless I quit IKEMEN GO, the game will repeat forever.
